tests/test_runtime.py

```diff
@@ -132,10 +132,9 @@ async def error(*args, **kwargs):
 
         assert self.runtime.state == RuntimeStates.error, self.runtime.state
 
-    @pytest.mark.skip(reason="https://github.com/valory-xyz/open-aea/issues/408")
+    # @pytest.mark.skip(reason="https://github.com/valory-xyz/open-aea/issues/408")
```

Comment:
Should delete this and close #408.

Or maybe you would like to make sure the flakiness is gone first?

Reply:
let it stay for a while. I want to be sure it works.
can not guarantee it's not flaky anymore, need some ci runs set to be checked

Reply:
let it stay for a while. I want to be sure it works.
can not guarantee it's not flaky anymore, need some ci runs set to be checked